cfme/tests/intelligence/reports/test_validate_chargeback_report.py

```diff
@@ -64,10 +64,30 @@ def new_credential():
 def vm_ownership(enable_candu, clean_setup_provider, provider):
     # In these tests, chargeback reports are filtered on VM owner.So,VMs have to be
     # assigned ownership.
+    vm_name = provider.data['cap_and_util']['chargeback_vm']
+
+    def _cu_vm_running():
+        if not provider.mgmt.does_vm_exist(vm_name):
+            pytest.skip("Skipping test, cu-24x7 VM does not exist")
+        if provider.mgmt.is_vm_running(vm_name):
+            return True
+        elif provider.mgmt.is_vm_stopped(vm_name) or \
+            provider.mgmt.can_suspend and provider.mgmt.is_vm_suspended(vm_name) or \
+                provider.mgmt.can_pause and provider.mgmt.is_vm_paused(vm_name):
+                provider.mgmt.start_vm(vm_name)
+
+        logger.debug("Sleeping 15secs...(current state: {}, needed state: running)".format(
+            provider.mgmt.vm_status(vm_name)
+        ))
+        return False
+
+    wait_for(_cu_vm_running, num_sec=360, delay=15)
+
+    if not provider.mgmt.is_vm_running(vm_name):
+        pytest.skip("Skipping test, cu-24x7 VM is not running")
```

Comment:
Redundant, as ``wait_for`` or ``provider.mgmt.start_vm`` above will fail if it's not running.